### DISCUSSION TITLE
Is it possible to run vs 2019 code analyze by command and not by gui?

### DISCUSSION THREAD

**Discussion by RachelXGanon** [dotnet/roslyn#56540] [Q&A] [answered]
I'm using vs 2019 enterprise,
thanks!!

**Comment by jmarolf** [comment, marked as answer]
running `dotnet build` will run analyzers that you want to include in your build. See docs [here](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-files) for how to configure analysis.

In the .NET 6 there is the `dotnet format` command that can also be used to run analyzers and apply codefixes (offical docs [here](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-format#analyzers), some additional info [here](https://github.com/dotnet/format/blob/9629a48a9cfa777b98264a8eeb7b6736f75ce5eb/docs/3rd-party-analyzers.md))
